testsuite/Makefile

```diff
@@ -266,24 +268,24 @@ clean:
 
 .PHONY: report
 report:
-	@if [ ! -f _log ]; then echo "No '_log' file."; exit 1; fi
-	@awk -f makefiles/summarize.awk <_log
+	@if [ ! -f $$(TESTLOG) ]; then echo "No $$(TESTLOG) file."; exit 1; fi
```

Comment:
The use of `$$(TESTLOG)` here is a minor mistake that caused error messages to show in CI outputs:

```
/bin/sh: TESTLOG: command not found
```

I just fixed it in trunk ( 2273d2985ad3fa36d107c5d580c63ff902760136 ).